GVFS/GVFS.Common/InstallerRunPreCheckerBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace GVFS.Common
+{
+    public abstract class InstallerRunPreCheckerBase

```

Comment:
Why did you need to create a base class? Is that because of any limitation with `Mock`?